extensions.yaml

```diff
@@ -77,3 +77,8 @@ extensions:
   cron_time: '00 02 * * *'
   ci: AppVeyor
   ci-user: mih
+ datalad_gooey:
```

Comment:
```suggestion
 datalad_gooey:
  apt_depends: [xvfb]
```
?

Reply:
wild guess that it might be needed since also listed for apt installs in .appveyor of gooey

Reply:
you would need to also regenerate that .github/workflows/test-datalad_gooey-maint.yaml file now. configuration is read to generate that file